examples/README.md

```diff
@@ -20,6 +20,7 @@ Container engines:
 - [`docker.yaml`](./docker.yaml): Docker
 - [`podman.yaml`](./podman.yaml): Podman
 - [`singularity.yaml`](./singularity.yaml): Singularity
```

Comment:
Can we now remove singularity? Or maybe move to the `deprecated` directory

Reply:
This new "apptainer" package is not yet available in any distributions, though.

Reply:
Requested a package for aarch64, currently the _only_ architecture is x86_64

Reply:
I moved it to `experimental`, and then removed the arm64 arch from list (for now)